modules/ROOT/pages/deployment/general/roles-permissions.adoc

```diff
@@ -0,0 +1,107 @@
+:toc: right
+
+:description: There are two general aspects with regards to roles and permissions. These aspects address  accounts, groups and spaces from a backend perspective and the same from a frontent perspective. The way how this has been implemented makes that concept open for upcoming and extended features.
+
+== Introduction
+
+{description}  
+
+== General Info
+
+When it comes to roles and permissions, there is a big difference between the backend (Infinite Scale) and the frontent (ownCloud Web). This is because Infinite Scale provides capabilities with the backend that can be used by using API's and a frontend that implements best practices for users by using these API's.
+
+This means, that the backend currently provides much more capabilities than the frontend offers to users. 
+
+For the moment, it would be counter acting to provide methods to users or admins via the web interface which are not fully developed and therefore could potentially harm the system. For these reasons, restrictions have been implemented in the frontend to protect the Infinite Scale backend.
+
+Examples where the frontent currently provides limited capabilities to users though the backend has more capabilities:
+
+* Admins, Space Admins and Users do not have full configuration and managability via the web interface.
+* Admins and Space Admins can currently only view Spaces that have been created by them selves.
+* Users cant create Spaces on their own.
+
+Over time, methods will be implemented in the frontend to provide capabilities the backend provides in the best and most user friendly way.
+
+== Backend Roles and Permissions
+
+Currently there are four predefined backend roles which have fixed assigned permissions where each permission has grants specific to a role. To be noted, that some grants are given only for the account itself and not in general.
+
+Consider that these roles and permissions are provided by the backend via API's which means you can always call them by own code or via `curl`. 
+
+Read the image below the following way::
+* A Role gets one or more permission types assigned
+* Each permission type is granted individual rights relevant for that role
+* Roles are assigned to accounts
+
+// the empty macro is necessary to have more distance between text and image
+ 
+{empty} +
+
+image::deployment/general/roles-and-permissions.svg[width=700]
+
+=== Backend Role Details
+
+In general with regards to Spaces, these can always be deleted by the creator and do not need a special permission assignment with the exeption of the `Delete Home Space` permission, which is only assigned to the Admin.
+
+Admin::
+The `Admin` role is in full control of all permission types with one exeption which is `Self Management`. This permission type is excluded to avoid that the last admin accidentially reduces his own role to a lower one making the system unmanagable. As long there is more than one admin, roles can be set by the `Role Management` permission.
```

Comment:
```suggestion
The `Admin` role is in full control of all permission types with one exception which is `Self Management`. This permission type is excluded to avoid that the last admin accidentally reduces their own role to a lower one making the system unmanageable. As long as there is more than one admin, roles can be set by the `Role Management` permission.
```